deployments/multiple-ragapps/.env

```diff
@@ -1,4 +1,8 @@
 # Path to the `data` directory which keeps the state
 # Must be an absolute path on Windows, e.g.:
 # STATE_DIR=C:\absolute\path\to\your\data
-STATE_DIR=./data
+STATE_DIR=
```

Comment:
`./data` don't work on Linux. I think we should required user to set  STATE_DIR to the current directory.